policyengine_us/parameters/gov/states/va/tax/income/subtractions/subtractions.yaml

```diff
@@ -20,6 +20,8 @@ metadata:
   period: year
```

Comment:
Could you take a look at this line:
`    - military_service_income # Certification Number 28
`
Military service income as used as national guard pay and I think we subtract all of it but it should be capped - even before 2023 
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/86579b60-50ef-405d-8a19-e28f3c2a8949)